ui/src/App.tsx

```diff
@@ -22,6 +28,44 @@ const TABS: TabsType[] = ['today', 'yesterday', 'archived'];
 function App() {
   const [tab, setTab] = useState<TabsType>('today');
 
+  function CreateButton(i: number){
```

Comment:
```suggestion
  const GridButton = () => {
```

Traditional function syntax has largely been replaced by lambdas: https://www.vinta.com.br/blog/javascript-lambda-and-arrow-functions

Also, generally components are moved into their own file, i.e. `GridButton.tsx` and then imported.